src/test/resources/schemas/description.json

```diff
@@ -17,6 +17,9 @@
         "description": "A string indicating the type of description",
         "type": "string"
       }
-    }
+    },
+    "required": [
+      "label"
```

Comment:
Are there cases where the is only a `label` and no `id`? Should there be?

Reply:
You are correct, adjusting the schema.